src/OctoshiftCLI.Tests/Octoshift/Services/GithubApiTests.cs

```diff
@@ -3368,12 +3368,52 @@ public async Task AbortMigration_Returns_True_On_Success()
             .ReturnsAsync(response);
 
         // Act
-        var expectedBooleanResponse = await _githubApi.AbortMigration(migrationSourceId);
+        var expectedBooleanResponse = await _githubApi.AbortMigration(migrationId);
 
         // Assert
         expectedBooleanResponse.Should().Be(actualBooleanResponse);
     }
 
+    [Fact]
+    public async Task AbortMigration_Surfaces_Error_With_Incorrect_Migration_ID()
+    {
+        // Arrange
+        const string migrationId = "1234";
+
+        const string query = @"
+                mutation abortRepositoryMigration(
+                    $migrationId: ID!,
+                )";
+        const string gql = @"
+                abortRepositoryMigration(
+                    input: { 
+                        migrationId: $migrationId
+                    })
+                   { success }";
+
+        var payload = new
+        {
+            query = $"{query} {{ {gql} }}",
+            variables = new
+            {
+                migrationId,
+            },
+            operationName = "abortRepositoryMigration"
+        };
+
+        const string actualErrorResponse = $"Invalid migration id: {migrationId}";
```

Comment:
nit: this is the expected message not actual a more appropriate name would be: `expectedErrorMessage`.